backend/src/bundles/achievements/achievement.controller.ts

```diff
@@ -0,0 +1,301 @@
+import { type AchievementService } from '~/bundles/achievements/achievement.service.js';
+import { type UserAuthResponseDto } from '~/bundles/users/users.js';
+import {
+    type ApiHandlerOptions,
+    type ApiHandlerResponse,
+} from '~/common/controller/controller.js';
+import { BaseController } from '~/common/controller/controller.js';
+import { ApiPath } from '~/common/enums/enums.js';
+import { HttpCode } from '~/common/http/http.js';
+import { type Logger } from '~/common/logger/logger.js';
+
+import {
+    type UserGetAllItemResponseDto,
+    AchievementsApiPath,
+} from './enums/enums.js';
+
+/**
+ * @swagger
+ * components:
+ *   schemas:
+ *     Error:
+ *       type: object
+ *       properties:
+ *         errorType:
+ *           type: string
+ *           enum:
+ *              - COMMON
+ *              - VALIDATION
+ *         message:
+ *           type: string
+ *
+ */
```

Comment:
let's move this declaration to jenerateDoc() in base-server-app-api already, since we need to declare it in every controller.

Reply:
Thank you, I have already fix it